frontend/app/views/joiner/form/payment.scala.html

```diff
@@ -78,32 +81,34 @@ <h2 class="form-group__title">Address</h2>
                     </div>
 
-                    @if(plans.tier != Supporter()){
-                        <div class="form-group">
+                    @if(plans.tier != Supporter()) {
+                        <div class="form-group form-section__promotion">
                             <h2 class="form-group__title">Promotion</h2>
                             @fragments.form.promoCode(trackingPromoCode, promoCodeToDisplay)
                         </div>
                     }
 
-                    <div class="form-group">
-                        <h2 class="form-group__title">Billing</h2>
-                        @fragments.form.billingAddress("Billing address")
-                        @fragments.form.addressDetail(
-                            countriesWithCurrencies = countriesWithCurrencies,
-                            formType = "billingAddress",
-                            addressRequired = true,
-                            address1 = idUser.privateFields.billingAddress1,
-                            address2 = idUser.privateFields.billingAddress2,
-                            town = idUser.privateFields.billingAddress3,
-                            postcode = idUser.privateFields.billingPostcode,
-                            county = idUser.privateFields.billingAddress4
-                        )
+                    <div class="form-group form-group--little-padding">
+                        <div class="form-section__billing">
+                            <h2 class="form-group__title">Billing</h2>
+                            @fragments.form.billingAddress("Billing address")
+                            @fragments.form.addressDetail(
+                                countriesWithCurrencies = countriesWithCurrencies,
+                                formType = "billingAddress",
+                                addressRequired = true,
+                                address1 = idUser.privateFields.billingAddress1,
+                                address2 = idUser.privateFields.billingAddress2,
+                                town = idUser.privateFields.billingAddress3,
+                                postcode = idUser.privateFields.billingPostcode,
+                                county = idUser.privateFields.billingAddress4
+                            )
+                        </div>
                         @fragments.form.paymentOptions(plans)
                         @fragments.form.cardDetail(plans)
                     </div>
-
+                    @fragments.form.terms(countryGroup)
                     @fragments.form.errorMessageDisplay()
-                    @fragments.form.submit(PaidPlans(plans), countryGroup = countryGroup, dataCategory = Some("checkout page"), dataLabel = Some("payment a-annual"))
+                    @fragments.form.submit(PaidPlans(plans), membershipTermsAndConditions = false, countryGroup = countryGroup, dataCategory = Some("checkout page"), dataLabel = Some("payment b-annual"))
```

Comment:
It is been used in other files e.g. ***freeToPaid.scala.html***, ***addressWithWelcomePack.scala.html***, etc.